modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -1,1190 +1,1167 @@
+| Socket Timeout a| Number |  The amount of time to wait (in milliseconds) for data to be transferred over an established, open connection before the connection times out. A value of 0 means infinity and is not recommended. |  50000 | 
+| Access Key a| String |  The access key provided by Amazon. |  | x
+| Secret Key a| String |  The secret key provided by Amazon. |  | x
+| Try Default AWSCredentials Provider Chain a| Boolean |  Set this field to true to obtain credentials from the AWS environment. See the Amazon https://docs.aws.amazon.com/sdk-for-java/v2/developer-guide/credentials.html[credentials] topic. |  false | 
+| Max Connections a| Number |  Sets the maximum number of allowed open HTTP connections. |  | 
+| Region Endpoint a| String |  Set the topic region endpoint. |  US_EAST_1 | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  | 
+| Host a| String |  The optional proxy host. |  | 
+| Port a| Number |  The optional proxy port. |  | 
+| Username a| String |  The optional proxy username. |  | 
+| Password a| String |  The optional proxy password. |  | 
+| Domain a| String |  The optional proxy domain. |  | 
+| Workstation a| String |  The optional proxy workstation. |  | 
+|===
+
+== Supported Operations
+* <<describeStream>> 
+* <<getRecords>> 
+* <<getShardIterator>> 
+* <<listStreams>> 
+
+
 
 == Operations
 
 [[batchDeleteItem]]
-== Batch Delete Item
-
+=== Batch Delete Item
 `<dynamodb:batch-delete-item>`
 
-=== Parameters
 
-[%header%autowidth.spread]
+FIXME: Add docs.
+
+
+==== Parameters
+[%header,cols="20s,20a,30a,20a,10a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Request Delete Items a| Object |  One or more items to delete from table. |  |x
-| Target Variable a| String |  The name of a variable where the output of this operation is stored |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy in case of connectivity errors |  |
+| Configuration | String | The name of the configuration to use. | | x
+| Request Delete Items a| Object |  One or more items to delete from table. |  | x
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` | 
```

Comment:
An expression to evaluate against the operation's output. The expression's outcome is stored in the target variable.